[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/core/overview-of-the-bam-development-process.md

1.  Create an observation model with the BAM Add-in for Excel.

    > [!NOTE]

>     >  Examples of the steps noted in this procedure can be found in the BAM API (BizTalk Server sample), [here](/biztalk/core/bam-api-biztalk-server-sample).

Reviewer: ```suggestion
    > You can find examples that show the steps in this procedure in the [BAM API BizTalk Server sample](/biztalk/core/bam-api-biztalk-server-sample).
```
